### PR TITLE
chore(ui): Editor textfield visual fixes

### PR DESCRIPTION
**Pull request by bartusek27**
## Bug

- [x] Related issues linked using closes: #1738

- Removed outline of overlaying textfield above slate and aligned it
- Let the modal windows of medical records to be auto-width

### CODE REVIEW

File: apps/client/src/components/editor/editor.tsx [hunk 1]
```diff
@@ -141,7 +141,7 @@ export function Editor(props: EditorProps) {
           renderLeaf={renderLeaf}
           renderElement={renderElement}
           className={classNames(
-            "w-full bg-transparent disabled:cursor-not-allowed disabled:opacity-80 py-1.5",
+            "w-full bg-transparent disabled:cursor-not-allowed disabled:opacity-80 outline-none",
```
Comment (Dev-CasperTheGhost) [inline apps/client/src/components/editor/editor.tsx:144]:
```suggestion
            "w-full bg-transparent disabled:cursor-not-allowed disabled:opacity-80 outline-none",
            props.value.length <= 3 ? "pb-10" : "py-1.5",
```

File: apps/client/src/components/editor/editor.tsx [hunk 2]
```diff
@@ -113,7 +113,7 @@ export function Editor(props: EditorProps) {
   return (
     <div
       className={classNames(
-        "mt-1 rounded-md border w-full shadow-sm",
+        "mt-1 rounded-md border w-full shadow-sm pb-10",
```
Comment (Dev-CasperTheGhost) [inline apps/client/src/components/editor/editor.tsx:116]:
```suggestion
        "mt-1 rounded-md border w-full shadow-sm",
```

**Review by Dev-CasperTheGhost — CHANGES_REQUESTED** [review]
Minor fixes :)

**Review by Dev-CasperTheGhost — APPROVED** [review]

File: apps/client/src/components/editor/editor.tsx (hunk 1 above)
Comment (bartusek27) [reply apps/client/src/components/editor/editor.tsx:144]: Dunno why, but okey :)

**Pull request merged**